### DISCUSSION TITLE
Hashed-Password not working

### DISCUSSION THREAD

**Discussion by TheGreatestJannet** [coder/code-server#2666] [Q&A] [answered]
Hello, I have set my hashed password in `config/.config/code-server/config.yaml`. I have set: `hashed-password: hashedpassword`. However at startup I get `starting with no password` and no password is asked for. Anyone know why? I'm running the linux-server docker image on a Raspberry Pi 4.

**Comment by TheGreatestJannet** [comment]
In fact even using `- PASSWORD=PASSWORD` in the docker-compose file doesn't work.

**Reply by TheGreatestJannet** [reply]
Recreating the container with a password set in the docker-compose file seems to work but it still won't pick up the hashed password from the config.yaml even after removing the password set in the docker-compose file.

**Comment by TheGreatestJannet** [comment, marked as answer]
Fixed. It seems you need to use `- PASSWORD=PASSWORD` as well as `- HASHED_PASSWORD` with the linuxserver container

**Reply by tim-rohrer** [reply]
Doesn't that defeat the purpose of using `HASHED_PASSWORD`?
